### PR TITLE
[OSDOCS-10434]:Update Standard Instance Types Based https://issues.redhat.com/browse/OCM-6489

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.15+
Issue:

https://issues.redhat.com/browse/OSDOCS-10434
Link to docs preview:

https://75455--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_planning/rosa-limits-scalability.html#node-sizing-during-installation_rosa-limits-scalability _
https://75455--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_planning/osd-limits-scalability#node-sizing-during-installation_osd-limits-scalability
QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by svmrh** [comment]
The changes made in 'GCP control plane and infrastructure node size' table under "Node sizing during installation" section looks good. https://75455--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_planning/osd-limits-scalability#node-sizing-during-installation_osd-limits-scalability. But please note:

- OCM-6489 changes for CCS are currently in final stages. pending:
   - OCM-7607 is in review i.e. pending QE testing
   - PERFSCALE-2910 track validating OSD-GCP scaling numbers with standard GCP instances on controlplane and infra nodes
   - Removing the OCM backend feature gate. 

Once above is completed, https://75455--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_planning/osd-limits-scalability#node-sizing-during-installation_osd-limits-scalability changes can go live. 

Side note: 'GCP control plane and infrastructure node size' table change be updated for non-CCS once the following changes for supporting standard GCP instances for controlplane and infra nodes for non-CCS are complete: 
   - [OCM-6726](https://issues.redhat.com/browse/OCM-6726) - AMS quota changes for supporting standard GCP instances (E2 and C2) in non-CCS deployment
   - [OCM-7631](https://issues.redhat.com/browse/OCM-7631) - QE validating standard GCP instances (E2 and C2) in non-CCS deployment
   - then comes, OCM changes for supporting standard GCP instances for control and infra nodes in non-CCS deployment (similar to OCM-6489)
   - Validating the impact of standard GCP instances on the overall scaling numbers in non-CCS deployment (PERFSCALE-2910)

**Comment by svmrh** [comment]
Hi Mark,
As discussed, we need slight modification to how we publish the node types and sizes used by control plane and infrastructure nodes. Instead of adding a superscript "[1] CCS-only", add a new table with this text below the existing one to highlight the new node types

**“OpenShift Dedicated clusters created on Google Cloud Platform (GCP) after __<to-be-added-later>__, 2024 using Customer Cloud Subscription (CCS) billing model will use the following GCP control plane and infrastructure node sizes”**
_(feel free to modify the above text as you feel apporpriate)_

| Number of compute nodes      | Control plane node size | Infrastructure node size     |
|  :----:             |    :----:   |             :----: |
| 1-25      | n2-standard-8       | n2-highmem-4   |
| 26-100   | n2-standard-16        | n2-highmem-8      |
| 101-180   | n2-standard-32        | n2-highmem-16      |

**Comment by mletalie** [comment]
Hello @svmrh, thanks for that information. Please see current iteration. I think what I have included for informational content is in line with what we currently have for the preceding tables while being specific enough for the end user. Please let me know what you think when you get a chance. Appreciate it!

**Review by JakobGray — APPROVED** [review]

**Comment by mletalie** [comment]
FYI reviewers date format based on IBM styleguide https://www.ibm.com/docs/en/ibm-style?topic=measurement-dates-times

File: modules/sd-planning-considerations.adoc [hunk 1]
```diff
@@ -68,6 +70,26 @@ GCP control plane and infrastructure node size
 |custom-16-131072-ext
 |===
 
+GCP control plane and infrastructure node size for clusters created on or after 6 June 2024.
+[options="header",cols="2a,2a,2a"]
+|===
+|Number of compute nodes
```
Comment (obrown1205) [inline modules/sd-planning-considerations.adoc:76]: There should be a colon after each example sentence (I think). See https://spaces.redhat.com/display/CCSENABLE/Colons+and+semicolons ("Use a colon after the label of a note.") Thoughts?

Number of compute nodes:
Control plane size:
Infrastructure node size:

**Comment by obrown1205** [comment]
Just a small nit, otherwise LGTM! Thank you for your thoroughness :)

File: modules/sd-planning-considerations.adoc (hunk 1 above)
Comment (mletalie) [reply modules/sd-planning-considerations.adoc:76]: Hi @obrown1205,
    Hope all is well! A bit confused by what you are asking/suggesting: Are you saying we should have a colon after each one of the sentences (shown in the attached image)? I don't think we should have them after the header titles. 
But, you bring something to my attention regardless. I put a period after the sentence I contributed to the docs, and the two leading sentences for the tables above do not have any punctuation, so I do need to match those sentences. Maybe add a colon to all three? 
  
image

Comment (obrown1205) [reply modules/sd-planning-considerations.adoc:76]: Yep! Sorrty - that's what I was referring to. Added the note to the wrong sentences :D

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-in-progress

**Comment by mletalie** [comment]
**FYI merge reviewers:** Not sure why the `peer-review-in-progress` label is still showing?

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-needed

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Comment by mletalie** [comment]
> Merge review LGTM.

Thank you, Sir!

**Pull request merged**